test/PR_test/unit_test/dataset/test_data.py

```diff
@@ -41,7 +41,9 @@ def test_dataset_urls(self):
             if isinstance(value, list):
                 for url in value:
                     with self.subTest('{}{} url'.format(key, url)):
-                        self.assertTrue(ping(url))
+                        if not ping(url):
+                            print("Warning!!: {} is not reachable at {}".format(key, url))
```

Comment:
Is there something with more oomf than a print message that we can do here?

Reply:
like print("WARNING!!!!!!!!!!!!!!!") ?

Reply:
lol. like something that would show up in the test summary as neither passed nor failed.

Reply:
The warning is in colored font now, pretty noticeable.  Didn't find anything that can show up in the end tho.